lib/platform/gitlab/index.ts

```diff
@@ -658,7 +658,10 @@ export async function findIssue(title: string): Promise<Issue | null> {
 
 export async function ensureIssue(
   title: string,
-  body: string
+  body: string,
+  once = false,
+  reopen = true,
+  labels: string[] = []
```

Comment:
@ViceIce we should probably soon convert function calls like this to receive an object, right?

Reply:
@rarkins  yes, this would make extending more easy. I would like to transform all js to ts first, then we can unify some interfaces. Also refactoring multiple function args to object spread is easier.